vmdb/spec/models/provider_foreman_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+
+describe ProviderForeman do
+  let(:provider) do
+    FactoryGirl.build(:foreman_provider,
+                      :url      => "example.com",
```

Comment:
since we are testing these values, I like to pass it in instead of relying upon a value in the factory.

I have set a default in that provider